src/routes/enemy.js

```diff
@@ -30,6 +30,19 @@ module.exports = function(mongoose) {
         });
     });
 
+    enemy.get('/delete/:nodename', (req, res) => {
```

Comment:
I would change `.get('/delete/:nodename',` to `.delete('/:nodename',` It doesn't matter too much but I think it more closely follows RESTful API conventions. I tested it and everything worked fine so you're good to merge.